tooling/operation/get-parameters-as-json-schema.js

```diff
@@ -322,11 +328,20 @@ function constructSchema(data, prevSchemas = [], currentLocation = '') {
   ['allOf', 'anyOf', 'oneOf'].forEach(polyType => {
     if (polyType in schema && Array.isArray(schema[polyType])) {
       schema[polyType].forEach((item, idx) => {
-        schema[polyType][idx] = constructSchema(item, prevSchemas, `${currentLocation}/${idx}`);
+        schema[polyType][idx] = constructSchema(item, prevSchemas, `${currentLocation}/${idx}`, jwtDefaults);
       });
     }
   });
 
+  // Check to see if defaults and currentLocation exist and it's not just an empty object and blank
+  // If currentLocation is blank, jsonPointer returns the whole object so it will shove everything into formData
```

Comment:
Care to slightly rewrite this to explain JWT defaults in here and why we might want to add them as a schema default? Also if you can wrap variables in backticks like I've done in other code comments in this file that'd be awesome.